cumulusci/core/config/OrgConfig.py

```diff
@@ -63,6 +67,26 @@ def _refresh_token(self, keychain, connected_app):
     def lightning_base_url(self):
         return self.instance_url.split(".")[0] + ".lightning.force.com"
 
+    @property
+    def salesforce_client(self):
+        if not self._client:
+            self._client = Salesforce(
```

Comment:
Use cumulusci.salesforce_api.utils.get_simple_salesforce_connection to get the simple-salesforce client, to make sure we're consistent in how it gets set up.

Reply:
That function wants a project_config. Will we always have it whenever we want to grab a salesforce_client from an org? The OrgConfig object doesn't seem to have a reference to the project_config so it would need to be passed from the client. Or get_simple_salesforce_connection could be refactored to make the project_config optional.

Reply:
@prescod sorry, I was thinking about a different context. I'd like to adjust get_simple_salesforce_connection to do something reasonable when a project_config isn't available, but we can do that separately.